lsif/src/server/middleware/errors.ts

```diff
@@ -9,18 +15,22 @@ import { pipeline as _pipeline } from 'stream'
  */
 export const errorHandler = (
     logger: Logger
-): ((error: any, req: express.Request, res: express.Response, next: express.NextFunction) => void) => (
-    error: any,
+): ((error: unknown, req: express.Request, res: express.Response, next: express.NextFunction) => void) => (
+    error: unknown,
     req: express.Request,
     res: express.Response,
+    // Express uses argument length to distinguish middleware and error handlers
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     next: express.NextFunction
 ): void => {
-    if (!error || !error.status) {
-        // Only log errors that don't have a status attached
+    const status = (isApiError(error) ? error.status : 500) || 500
```

Comment:
You can save the redundant 500
```suggestion
    const status = (isApiError(error) && error.status) || 500
```